recipes-domx/meta-xt-images-vgpu/recipes-graphics/gles-module/gles-module-egl-headers_1.7.bb

```diff
@@ -0,0 +1,4 @@
+require gles-module-egl-headers.inc 
+
+# dummy path through...
```

Comment:
just leave include only w/o any comment. the reason you need it is clear as version is a part of the recipe name